syntax = "proto3";

package mainpb;

message UserData {
    optional bool status = 1;
    optional string info = 2;
    string firstName = 3;
    string lastName = 4;
    optional string middleName = 5;
    string email = 6;
    uint32 rating = 7;
    uint32 role = 8;
    optional string telegramLink = 9;
    optional string githubLink = 10;
    optional string stepikLink = 11;
    optional string kaggleLink = 12;
    string avatarPath = 13;
}

message AuthData {
    string email = 1;
    string password = 2;
}

message AccessToken {
    string accessToken = 1;
    string tokenType = 2;
}
